.github/pull.yml

```diff
@@ -0,0 +1,7 @@
+version: "1"
+rules:
+  - base: main
+    upstream: cs-pub-ro:main
+    mergeMethod: none
+
```

Comment:
Please add @teodutu as assignee and @gabrielmocanu as reviewer as they mention in the documentation.